### DISCUSSION TITLE
Kudos to Shaun - Awesome work, big thanks!

### DISCUSSION THREAD

**Discussion by iJungleboy** [oqtane/oqtane.framework#1795] [Ideas]
Hey @sbwalker 

Just wanted to let you know I really admire your work and efforts here - and the hundreds of hours you spend in making this possible. Thank you so much!

...and for everybody else - feel free to share some thanks and compliments here, Shaun and the others really deserve it!

**Comment by lanthonyneville** [comment]
Me too. What I really see is the commitment to enforce strong modern architectural principles that are flexible enough to allow for diverse use cases .... all the lessons learnt from the DNN days I guess ... the Benevolent Dictator 2.0 you could say :)

**Reply by Loclhh** [reply]
Me too! I don't know how to express my thankful to Shaun for all things he did for us! Very appreciated!

**Reply by Raceeend** [reply]
Me too. To Shaun and all other contributors chapeau / hats off

**Comment by thienvc** [comment]
I look forward to each day that oqtane is big enough to switch from dnn, actually I regret leaving dnn but I've been thinking a lot.

**Comment by wbonekamp** [comment]
I agree... Shaun did awesome work! DNN is still a good CMS to create websites in but Oqtane is the way to the future (.net core + blazor).
Hopefully Oqtane will become the all in one solution... and we also don't need ABP Framework anymore (at least for the smaller portal solutions)

DNN + ABP => Oqtane

Does anywone already used Oqtane for production portals / projects?
@sbwalker Do you think we can migrate DNN website to Oqtane by Q2/Q3 2022?

Thanks again!

**Reply by sbwalker** [reply]
@wbonekamp I believe we are already in Q4 2021 - did you mean Q2/Q3 2022? Either way, there is no simple answer to your question about being able to migrate a DNN website to Oqtane. Although DNN and Oqtane have some conceptual similarities in terms of capabilities, they are based on completely different technology, architecture, etc... So technically it is not possible to provide a full migration from DNN to Oqtane. That being said it would be possible to migrate some aspects... for example, it would certainly be possible to migrate the data from the core DNN database tables to Oqtane. This would allow for the migration of the sites, pages, modules, permissions, roles, users, etc... However it would not be possible to migrate any custom DNN extensions such as skins, modules, etc.... The only migration options for custom extensions is to recreate them using the new technology and architecture.

**Reply by zhengdayongbob** [reply]
Thanks @sbwalker 's great work, I already publiched my project to production to be used for my customer for 6 months. Everything works fine and the application works stable.

**Reply by zhengdayongbob** [reply]
I integrated MudBlazor based on Oqtane 1.0 version. this is a great framework, Will try to upgrade to the latest 3.0 framework.

**Reply by sbwalker** [reply]
@wbonekamp the oqtane.org website was migrated to Oqtane in Q1 2022. The same installation is also running www.dnfprojects.com and builtonblazor.org. The performance is impressive and my Azure hosting actually decreased by -$50/month

**Reply by wbonekamp** [reply]
Great and looking good! so we have the first production website running on Oqtane now... I noticed I'm not the only DNN developer looking for Oqtane making progress... We installed the latest version of Oqtane in our sandbox and started experimenting with it... performance won't be the problem...

**Reply by sbwalker** [reply]
@wbonekamp actually https://www,dnfprojects.com has been running Oqtane in production (on Azure) since July 2021. I was using it to dogfood every release for the past 9 months ;)

**Comment by ghhv** [comment]
Agreed too - this was a concept I had dreamt about but had no clue on where to begin. I've only just started this journey with Oqtane but hope to build big things.
